db/migrate/20170605090919_add_expression_type_to_miq_alert.rb

```diff
@@ -0,0 +1,23 @@
+class ::MiqExpression; end
+
+class AddExpressionTypeToMiqAlert < ActiveRecord::Migration[5.0]
+  class MiqAlert < ActiveRecord::Base
+    serialize :expression
+  end
+
+  def up
+    add_column :miq_alerts, :expression_type, :string
+
+    say_with_time('Add expression type to MiqAlert for existing alerts') do
+      MiqAlert.all.each do |alert|
+        alert.expression_type = Hash if alert.expression.kind_of?(Hash)
+        alert.expression_type = MiqExpression if alert.expression.kind_of?(MiqExpression)
```

Comment:
what is this expression_type in SQL?  A string?  If so, I expect above 2 lines to set it this value to a string, and not a Ruby class.